### PR TITLE
I added my folder!

### PR DESCRIPTION
**Pull request by WhiteOlivierus**
It wouldn't do the folder alone, so there is also the base of the first homework assignment.

### CODE REVIEW

**Comment by WhiteOlivierus** [comment]
Yes, because I pushed the file before editing because it wouldn't push only the folder.  It will be changed in the next push.

File: Bastien Olivier Dijkstra/Clean.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+public class Clean : MonoBehaviour
+{
+
+    public float range = 100f;
+
+    private RaycastHit selected;
+    private RaycastHit lastSelected;
+
+    private AudioSource cleanSoundSource;
+
```
Comment (Jerstep) [inline Bastien Olivier Dijkstra/Clean.cs]: Missing the dubble white spaces (enters).

File: Bastien Olivier Dijkstra/Clean.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    void Start()
+    {
+        cleanSoundSource = GetComponent<AudioSource>();
+        Physics.Raycast(transform.position, transform.forward, out selected, range);
+        lastSelected = selected;
+    }
+
```
Comment (Jerstep) [inline Bastien Olivier Dijkstra/Clean.cs]: Your brackets are not yet on the right lines.

File: Bastien Olivier Dijkstra/Clean.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    void SelectMesh(Renderer [] toHighLight, bool select)
+    {
+        foreach(Renderer r in toHighLight)
+        {
+            if(select)
+                r.material.color = new Color(0, 0, 1);
+            else
+                r.material.color = new Color(1, 1, 1);
+        }
+
```
Comment (Jerstep) [inline Bastien Olivier Dijkstra/Clean.cs]: The spaces behind the variables are good (1, 1, 1)

File: Bastien Olivier Dijkstra/Clean.cs (hunk 1 above)
Comment (WhiteOlivierus) [reply Bastien Olivier Dijkstra/Clean.cs]: Thank you for reminding me of the rules, but this was the unedited version of my homework. It wouldn't push only a folder. But I just reformated my file, your welcome to take a look.

File: Bastien Olivier Dijkstra/Clean.cs [hunk 4]
```diff
@@ -0,0 +1,58 @@
+﻿using UnityEngine;
+
+public class Clean : MonoBehaviour {
+
+    public float range = 100f;
```
Comment (DanielBergshoeff) [inline Bastien Olivier Dijkstra/Clean.cs]: While your code looks good and the brackets are all in the right places now, I noticed you haven't made use of Properties, which is something you could take a look at.

Comment (WhiteOlivierus) [reply Bastien Olivier Dijkstra/Clean.cs]: FINE! :P

**Pull request merged**